src/movableCoord.js

```diff
@@ -667,22 +667,22 @@ MovableCoord에 대한 상세한 내용은 데모를 살펴보기 바란다.
 		},
 
 		// set up 'css' expression
-		_reviseOptions: function(options) {
+		_reviseOptions: function() {
 			var key;
+			var self = this;
 			$.each(["bounce", "margin", "circular"], function(i, v) {
```

Comment:
you're right. I suggest that we use es5 code. but, we don't decide it yet.
I refactor InfiniteGrid totally and I test it antecedently.